app/views/admin/users/_bid_history.html.erb

```diff
@@ -4,8 +4,8 @@
   <thead>
     <tr>
       <th scope="col">Auction</th>
-      <th scope="col">Start Date</th>
-      <th scope="col">End Date</th>
+      <th scope="col">Status</th>
+      <th scope="col">Skills</th>
```

Comment:
spoke with michael and he seemed to think it was Ok to replace start/end date with status in order to preserve minimal table width